AmplifyPlugins/Core/AWSPluginsCore/API/AppSyncErrorType.swift

```diff
@@ -12,7 +12,7 @@ public enum AppSyncErrorType: Equatable {
 
     private static let conditionalCheckFailedErrorString = "ConditionalCheckFailedException"
     private static let conflictUnhandledErrorString = "ConflictUnhandled"
-
+    private static let dynomoDBConditionalCheckFailed = "DynamoDB:ConditionalCheckFailedException"
```

Comment:
move this line above line 14, to group the conditional check failed strings together

Reply:
moved